packages/sources/coinpaprika-test/src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+export const DEFAULT_API_ENDPOINT = 'https://api.coinpaprika.com'
+export const PRO_API_ENDPOINT = 'https://api-pro.coinpaprika.com'
+
+export const customSettings = {
```

Comment:
We should also add the `API_ENDPOINT` config to custom settings so that it's configurable through env var. I think you could follow `coingecko` as an example

Reply:
This would also require changes where you prepare requests